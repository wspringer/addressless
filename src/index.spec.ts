import { describe, it, expect } from "vitest";
import { lookupAddress } from ".";

describe("lookupAddress", async () => {
  it("should allow you to lookup an address", async () => {
    const result = await lookupAddress("4147AZ", "5");
    expect(result).toMatchSnapshot();
  });

  it("should return null in case no address is found", async () => {
    const result = await lookupAddress("5211ER", "1000");
    expect(result).toBeNull();
  });
});
