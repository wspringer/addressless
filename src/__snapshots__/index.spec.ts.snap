// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`lookupAddress > should allow you to lookup an address 1`] = `
{
  "city": "Asperen",
  "district": "Asperen",
  "houseNumber": "5",
  "latitude": 51.88051803,
  "longitude": 5.11206571,
  "municipality": "West Betuwe",
  "neighbourhood": "Asperen Oude Kern",
  "postcode": "4147AZ",
  "province": "Gelderland",
  "street": "Stadswal",
}
`;
